base-theme/assets/css/button.scss

```diff
@@ -45,3 +45,7 @@ a.red-btn {
   cursor: pointer;
   white-space: nowrap;
 }
+
+button.btn-link.text-black {
+  color: black;
+}
```

Comment:
**Requested Change:** We also want `vertical-align: baseline`, which will align the button text with surrounding text.

**Suggestion:** Since the "textual" buttons might not always be black, I think it would be better to ditch the color in class name and instead inherit the color from parent:
```css
button.btn-link.text-button {
  color: inherit;
  vertical-align: baseline;
}
```

Reply:
That makes sense, I just set these on `button.btn-link` though as that's the bootstrap style for link buttons. If you don't do this Bootstrap will try and set it to bootstrap alert style colors.